CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ##  Unreleased
 - Fix gsub! usage for open office documents on a frozen string [580](https://github.com/roo-rb/roo/pull/580)
+- Add support for boolean values in open office files that were generated via Google Sheets [579](https://github.com/roo-rb/roo/pull/579)
```

Comment:
This new line should be end with link https://github.com/roo-rb/roo/pull/580

Reply:
Let me fix this up. Must have been mixed up in the merge

Reply:
@kakubin Have another look now - the other line in the changelog was wrong as well. Both were using the issues numbers and not the PR numbers.